### ISSUE TITLE
Detailed question about your paper

### ISSUE THREAD

**Issue by kwonmha** [snunlp/KR-BERT#3]
I read your paper and good to see that less data is needed to train the model.

I would like to know some details like the number of layers, input length, whether additional training objectives like SOP is used related to pretraining.
I tried to find the info on your paper but failed.
Let me know.
Thanks.

**Comment by snunlp** [comment]
Hello,
We used 12 layers, 12 attention heads, and 768 hidden units (just the same as the original BERT-base), the max input sequence length of 512, the Adam optimizer, the learning rate of 1e-4, without additional training objectives.
Thank you.

**Comment by kwonmha** [comment]
Thank you for reply!
